src/nccl_ofi_net.c

```diff
@@ -1180,6 +1180,19 @@ static ncclResult_t set_nic_props_default(int dev, struct fi_info *nic_prov,
 	props->guid = dev;
 
 #if (NCCL_VERSION_CODE >= NCCL_VERSION(2, 12, 0)) /* Support NCCL v2.12 */
+	/*
+	 * Sets intranode latency for EFA networks.
+	 *
+	 * This value is chosen by measuring all reduce latency for
+	 * different NCCL algorithms and using that to calculate intra node
+	 * latency based on NCCL's tuning algorithm.
+	 *
+	 * A few different values around this value were tried to see which
+	 * chose the correct algorithm (tree or ring) most times across
+	 * different message and cluster sizes.
+	 */
+	props->latency = 150;
```

Comment:
Does using an EFA-specific value make sense here?  The master branch should be for generic for any OFI provider, no?

Reply:
Yes, you are right! will remove it.